install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -125,7 +125,7 @@ rules:
       - create
 
   - apiGroups:
-      - extensions
+      - apps
```

Comment:
can we support both?

Reply:
I'm guessing not as the link to the diff shows that it only uses apps now.

Reply:
we can give perms for both in case controllers attached to this role gets from ext or apps